doc/source/cluster/kuberay/quickstart.md

```diff
@@ -0,0 +1,111 @@
+(kuberay-quickstart)=
+
+# Quickstart
```

Comment:
I've copy-pasted the content from the old `kuberay.md`.

Will revise in a follow-up.